aws_xray_sdk/ext/bottle/plugin.py

```diff
@@ -0,0 +1,105 @@
+from bottle import request, response, SimpleTemplate

```

Comment:
Can we change the name of the file to middleware.py ? (reason: similar file name between all supported frameworks)

Reply:
np